### PR TITLE
Created Newsletter DB Model and Endpoints

### PR DESCRIPTION
**Pull request by mukul-mehta**
Created endpoint to add newsletter and retrieve the latest newsletter. 

The date format used is: DD-MM-YYYY

### CODE REVIEW

File: app/main/service/newsletter_service.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"""for issue operations"""
```
Comment (thealphadollar) [inline app/main/service/newsletter_service.py]: should be newsletter

File: app/main/service/newsletter_service.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    @staticmethod
+    def getLatest():
+        try:
+            nl = Newsletter.query.order_by(Newsletter.upload_time.desc()).first()
+            return nl, 200
+
+        except BaseException:
+            LOG.error(
+                "Couldn't fetch articles. Please try again later", exc_info=True)
```
Comment (thealphadollar) [inline app/main/service/newsletter_service.py]: Couldn't fetch newsletters*

File: app/main/service/newsletter_service.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            response_object = {
+                'status': 'fail',
+                'message': 'Try again',
+            }
+            return response_object, 500
+        
+    @staticmethod
+    def add_newsletter(data):
+        try:
+            nl = Newsletter.query.filter_by(description=data.get('description')).first()
```
Comment (thealphadollar) [inline app/main/service/newsletter_service.py]: this should be the date that has been passed.

File: app/main/service/newsletter_service.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            if nl is not None:
+                response_object = {
+                    'status': 'Invalid',
+                    'message': 'Newsletter already present',
+                }
+                LOG.info(
+                    'Newsletter already present in database. Redirecting to home page')
```
Comment (thealphadollar) [inline app/main/service/newsletter_service.py]: It's an API, why do you need redirection?

File: app/main/service/newsletter_service.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                return response_object, 300
+
+            nl = Newsletter(data.get('description'), data.get('publish_date'), data.get('newsletter_content'), data.get('cover_image_url'))
+
+            response_object = {
+                'status': 'Success',
+                'message': 'Issue added Successfully',
```
Comment (thealphadollar) [inline app/main/service/newsletter_service.py]: newsletter added successfully

File: app/main/service/newsletter_service.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            }
+            return response_object, 200
+
+        except BaseException:
+            LOG.error(
+                "Couldn't add new article. Please try again later", exc_info=True)
```
Comment (thealphadollar) [inline app/main/service/newsletter_service.py]: Couldn't add new newsletter

**Comment by thealphadollar** [comment]
Also, please fix the Travis CI issue.

**Comment by mukul-mehta** [comment]
I'm done with the requested changes.

**Comment by thealphadollar** [comment]
@mukul-mehta Do you want to try to deploy this on the server?

**Comment by mukul-mehta** [comment]
@thealphadollar I've never worked on deployment before but yes, I'll do it.

**Comment by thealphadollar** [comment]
Exact instructions are given in the dropbox paper. Please be careful.

**Pull request merged**